zephyr/prj.conf

```diff
@@ -38,4 +38,11 @@ CONFIG_NORDIC_QSPI_NOR=y # configuration options for MX25R64 flash device
 CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
 
 # max contacts, that can be stored
-CONFIG_ENS_MAX_CONTACTS=65536
\ No newline at end of file
+CONFIG_ENS_MAX_CONTACTS=65536
+
+CONFIG_LOG=y
+CONFIG_LVGL=y
```

Comment:
Please move them to native_posix_64.conf or another display overlay configuration.